css/style.css

```diff
@@ -1,14 +1,23 @@
+/* Font import */
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,700i');
 .static{
```

Comment:
did you have a merge conflict?

Reply:
No. The last one I sent was for assignment 1 this pull is for assignment 2